libraries/terminal/src/AnsiEscape.ts

```diff
@@ -32,6 +32,10 @@ export class AnsiEscape {
   private static readonly _backslashNRegExp: RegExp = /\n/g;
   private static readonly _backslashRRegExp: RegExp = /\r/g;
 
+  public static getEscapeSequenceForAnsiCode(code: number): string {
```

Comment:
This function does NOT receive an "ANSI code" as input.

It receives a single "attribute" of a Set Graphic Rendition (SGR) sequence, which is just one particular kind of [CSI sequence](https://en.wikipedia.org/wiki/ANSI_escape_code#CSI_(Control_Sequence_Introducer)_sequences), which is a subset of ANSI escape codes.  SGR sequences may also contain more than one such attribute.

I think we should avoid exposing a public API for manipulating the details of such codes.  The standard is very complex, and its availability across different consoles is wildly inconsistent.  The point of our APIs is that casual engineers would do best to avoid getting involved in computing such codes, and instead rely on a library to handle it.